benchmark/integration/tpcc_benchmark.cpp

```diff
@@ -59,6 +56,8 @@ class TPCCBenchmark : public benchmark::Fixture {
   storage::GarbageCollectorThread *gc_thread_ = nullptr;
   const std::chrono::milliseconds gc_period_{10};
   const std::chrono::milliseconds metrics_period_{100};
+
+  const char *logfile_ = (std::getenv("LOGFILE_NAME") == nullptr ? "/tmp/tpcc.log" : std::getenv("LOGFILE_NAME"));
```

Comment:
I will fix. I think I have to start a review first to push into your branch?

Reply:
The changes on your branch look good to me... do I have to do anything for you to push into this branch?

Reply:
No I got it.